Code/Editor/Controls/ConsoleSCB.h

```diff
@@ -136,6 +136,7 @@ class ConsoleVariableEditor
     ConsoleVariableModel* m_model;
     ConsoleVariableItemDelegate* m_itemDelegate;
     CVarBlock* m_varBlock;
+    static AZ::ConsoleCommandInvokedEvent::Handler m_commandInvokedHandler;
```

Comment:
This shouldn't be static, but a member of the `ConsoleVariableEditor` class

Reply:
actually, it has to be static because it's called by ConsoleVariableEditor::RegisterViewClass which was already (and necessarily) static